src/3d/qgsshadowrenderingframegraph.h

```diff
@@ -122,7 +122,8 @@ class QgsShadowRenderingFrameGraph : public Qt3DCore::QEntity
     void setupShadowMapDebugging( bool enabled, Qt::Corner corner, double size );
     //! Sets the depth map debugging view port
     void setupDepthMapDebugging( bool enabled, Qt::Corner corner, double size );
-
+    //! Sets the size of the buffers used for rendering
+    void setSize( int width, int height );
```

Comment:
better to take QSize parameter, as we already have size() function returning QSize